test/org/sagebionetworks/bridge/services/UserProfileServiceImplTest.java

```diff
@@ -60,4 +60,10 @@ public void canUpdateUserProfile() {
         assertEquals(testUser.getUsername(), userProfile.getUsername());
         assertEquals("123-456-7890", userProfile.getPhone());
     }
+    
+    @Test
+    public void canRetrieveStudyParticipants() {
+        // All we an really do here is verify no error is thrown.
+        profileService.sendStudyParticipantRoster(testUser.getStudy());
+    }
```

Comment:
Will this send a real email? How can we ensure that all our test studies are configured properly (with an email address set to "bridge-testing@sagebase.org", or similar) to ensure we don't get hammered by AWS's spam block?